src/components/RecordAutocomplete/index.js

```diff
@@ -248,10 +248,10 @@ const RecordAutocomplete = (props) => {
           InputProps={{
             ...params.InputProps,
             endAdornment: (
-              <React.Fragment>
+              <>
```

Comment:
These were set to React.Fragment for readability

Reply:
this was autofixed to conform with the latest version of the eslint react plugin due to [this rule](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-fragments.md)